tx-source/dash_core.php

```diff
@@ -406,6 +407,7 @@
 $Definition['HTML Emails'] = 'HTML Emails';
 $Definition['HTTP Strict Transport Security (HSTS) Settings'] = 'HTTP Strict Transport Security (HSTS) Settings';
 
+$Definition['IgnoreListRevoke'] = 'Revoke <b>%s</b>\'s ignore list privileges?';
```

Comment:
It seems like all of these strings are in the frontend and not the dashboard. If that is the case then these go in site_core.